pkg/istio-agent/xds_proxy.go

```diff
@@ -693,6 +693,13 @@ func sendUpstream(upstream xds.DiscoveryClient, request *discovery.DiscoveryRequ
 
 // sendDownstream sends discovery response.
 func sendDownstream(downstream adsStream, response *discovery.DiscoveryResponse) error {
+	tStart := time.Now()
+	defer func() {
+		// This is a hint to help debug slow responses.
+		if time.Since(tStart) > 10*time.Second {
```

Comment:
This feels generally useful for responses that are less than 10s; maybe set this to a flag of some sort?

Reply:
I feel same when i add this log

Reply:
What impacts we have with slower responses? Isn't send time on istiod measure this?

Reply:
No impact, just a warning now.